error: ‘AT_NO_AUTOMOUNT’ undeclared

Hi All, 
I am not sure if this is the right place to post this , apologies in advance if it's not. 

I am using buildroot to build a linux environment for an FPGA platform.
It gets to the point where it attempts to build genimage which is where it's failing at:

```
 

        genimage 11
        =====

        prefix:                 /nobackup/user_g/tmp_build/host

        compiler:               /cad/apps/gnu/linux/x86_64/6/local/gcc/9.1.0/bin/gcc
        cflags:                 -O2 -I/nobackup/user_g/tmp_build/host/include  -pipe -Wall -Wextra -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wpointer-arith -Wsign-compare -Wchar-subscripts -Wstrict-prototypes -Wshadow -Wformat-security -Wtype-limits -Wno-unused-parameter -ffunction-sections -fdata-sections -fvisibility=hidden
        ldflags:                -L/nobackup/user_g/tmp_build/host/lib -Wl,-rpath,/nobackup/user_g/tmp_build/host/lib  -Wl,--as-needed -Wl,--gc-sections

        debug:                  no
        hide symbols:           yes
        libconfuse:             -L/nobackup/user_g/tmp_build/host/lib -lconfuse

>>> host-genimage 11 Building
PATH="/nobackup/user_g/tmp_build/host/bin:/nobackup/user_g/tmp_build/host/sbin:/cad/apps/gnu/linux/x86_64/6/local/gcc/9.1.0/bin:/home/rs/lintel:/cad/local/bin:/cad/div/bin:/usr/cadtools/local/bin:/usr/cadtools/bin:/cad/bin:/cad/apps/gnu/linux/x86_64/6/bin:/usr/local/bin:/usr/local/sbin:/usr/sbin:/sbin:/usr/bin:/bin" PKG_CONFIG="/nobackup/user_g/tmp_build/host/bin/pkg-config" PKG_CONFIG_SYSROOT_DIR="/" PKG_CONFIG_ALLOW_SYSTEM_CFLAGS=1 PKG_CONFIG_ALLOW_SYSTEM_LIBS=1 PKG_CONFIG_LIBDIR="/nobackup/user_g/tmp_build/host/lib/pkgconfig:/nobackup/user_g/tmp_build/host/share/pkgconfig"  /usr/bin/make   -C /nobackup/user_g/tmp_build/build/host-genimage-11/
/usr/bin/make --no-print-directory all-am
  CC       genimage-genimage.o
  CC       genimage-config.o
  CC       genimage-util.o
  CC       genimage-crc32.o
  CC       genimage-image-cpio.o
util.c: In function ‘dir_size’:
util.c:648:36: error: ‘AT_NO_AUTOMOUNT’ undeclared (first use in this function); did you mean ‘S_AUTOMOUNT’?
  648 |   if (fstatat(fd,  d->d_name, &st, AT_NO_AUTOMOUNT) < 0) {
      |                                    ^~~~~~~~~~~~~~~
      |                                    S_AUTOMOUNT
util.c:648:36: note: each undeclared identifier is reported only once for each function it appears in
make[3]: *** [genimage-util.o] Error 1
make[3]: *** Waiting for unfinished jobs....
make[2]: *** [all] Error 2
make[1]: *** [/nobackup/user_g/tmp_build/build/host-genimage-11/.stamp_built] Error 2
make: *** [_all] Error 2

```

I am using `v11` apparently. 

Any suggestions welcome. 
I probably don't have enough environment information here so please let me know and I'll add it.

--- a/util.c
+++ b/util.c
@@ -33,6 +33,10 @@
 
 #include "genimage.h"
 
+#ifndef AT_NO_AUTOMOUNT
+#define AT_NO_AUTOMOUNT 0x800
+#endif
+
 static int loglevel(void)
 {
 	static int level = -1;

